nixos/modules/installer/cd-dvd/installation-cd-graphical-calamares-plasma5.nix

```diff
@@ -45,8 +45,8 @@
       ln -sfT ${pkgs.plasma5Packages.konsole}/share/applications/org.kde.konsole.desktop ${
         desktopDir + "org.kde.konsole.desktop"
       }
-      ln -sfT ${pkgs.calamares-nixos}/share/applications/io.calamares.calamares.desktop ${
-        desktopDir + "io.calamares.calamares.desktop"
+      ln -sfT ${pkgs.calamares-nixos}/share/applications/calamares.desktop ${
+        desktopDir + "calamares.desktop"
```

Comment:
Is this a deliberate upstream change? It seems odd for them to move away from the increasingly‐common reversed DNS format.

Please squash these changes into another commit to reduce the number of broken in‐between states.

Reply:
Okay, acknowledged. Though I wonder if there was any particular reason we were doing so.